package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "after-build": "pandoc dist/operating-agreement/index.html -o dist/operating-agreement/index.pdf",
     "build": "eleventy",
-    "start": "eleventy --serve"
+    "dev": "eleventy --serve"
```

Comment:
I changed this back, because it didn't make sense to me to change the name of the default run script to `start` in the restyle-with-tailwind branch.